submissions/best-of-its-genre-nanogamer7/origins.md

```diff
@@ -0,0 +1,46 @@
+## Best of its genre – Nanogamer7
+
+We've all heard of Factorios insanely [good ratings](https://steamdb.info/stats/gameratings/) – consistently rising to second place after Portal 2 again and again.[^1] Obviously many others will try to imitate this success in the automation genre, but where did Factorio get inspired from? Was it the first game solely designed for automation or merely the first one to do it successfully? That's what I'll be trying to answer in this first article of hopefully a multi‐part series.
+
+### Automation defined
+
+The first difficult question we'll have to answer though, would be what automation even is. In Cities:Skylines for example you are building your city, which then (theoretically) functions autonomously. The difference here is, your job isn't to automate – the cars spawn out of nothing, choose their destination, and get there by themselves, once you build roads – but to manage it. In Minecraft on the other hand, you are the one to make the automation even possible – just with pistons and hoppers, instead of inserters and belts. Comparing these two examples, I think you can loosely define the automation genre as *"making stuff that makes stuff"*.
+
+However that definition is still a bit vague: As someone who didn't play it, would Dwarf Fortress' gameplay of enabling dwarfs to *make stuff* would fall under my definition of automation? But let's not get to far from Factorio here, maybe someone else will explore this further in the future. *\*wink wink\**
+
+### There's a mod for it
+
+I've already touch on Minecraft as an example for automation, and it's popularity boom in 2013 and with it one of the largest modding communities ever[^2][^3] could just have made it the most popular example of automation. But that got me thinking: With mods in the likes of Buildcraft, IndustrialCraft, and Redpower featuring a very heavy focus on industrial machinery and automation, and release dates well before Factorio's development even started,[^4] could that have been an inspiration for Factorio?
```

Comment:
I think you can make this step to modding more gradual. You could say something like "automation is possible in vanilla Minecraft, but it does not even begin to compare to whats possible in modded Minecraft. Since it's boom in 2013 and one of the..."